kip-0031.md

```diff
@@ -0,0 +1,107 @@
+---
+KIP: "0031"
+Title: fungible-meta-v1
+Author: CryptoPascal  
+Status: Early Draft
+Type: Standard
+Category: Chainweb / Fungible
+Created: 2024-09-28
+---
+
+## Abstract
+
+Define and implement a new way for tokens creators to export their Meta-Data to Wallets, Frontends, and Other services.
+While this is currently handled off-chain, this proposal is intended to take a an "on-chain" approach.
+
+
+## Rationale
+
+While `fungible-v2` provides technical functions to allow "technical on-chain" interoperability between tokens, meta-data and information about fungible tokens are missing.
+
+Currently, listing tokens for Wallet, Frontends, Exchanges, and other third parties is a cumbersome process. Maintainers have to manually contact tokens issuers to retrieve logos, names, and other useful information to display to users. This ends up with disparities between different wallets or frontends. Users complain that they don't have reliable information about their investment.
+
+Moreover, token issuers have no clean and defined way to inform wallets when some changes happen with their token (Rebrand, new issuance, ...)
+
+And finally, since Kadena is multichains, wallets need to be informed on how to use the token, regarding chains deployment:
```

Comment:
```suggestion
And finally, since Kadena is consists of multiple chains, wallets need to be informed on how to use the token, regarding chain deployment:
```